tests/integration/session/test_sign_out_and_exit.py

```diff
@@ -95,7 +103,7 @@ def setUp(self):
     def test_sign_out_url(self):
         self.launchSurvey(schema_name="test_individual_response")
         self.saveAndSignOut()
-        self.assertInRedirect("census.gov.uk")
+        self.assertInRedirect("/signed-out")
```

Comment:
Is it okay if use `SIGNED_OUT_URL_PATH ` instead just for consistency

Reply:
Resolved.